src/config.py

```diff
@@ -29,7 +29,7 @@ class Config:
 class DevSettings(Settings):
     # Environment variables will take priority, so DATABASE_URL is overridden
     # when running on Docker because `localhost` cannot be used
-    database_url = "mysql+pymysql://dbuser:dbuser@localhost:3306/dnyf-group-db"
+    database_url = "mysql+pymysql://dbuser:dbuser@localhost:3306/dnyf-composite-db"
```

Comment:
I think you can delete this config param too

Reply:
Gotcha!